integ-test/src/test/java/org/opensearch/sql/ppl/GeoIpFunctionsIT.java

```diff
@@ -45,4 +70,86 @@ public void testGeoPluginInstallation() throws IOException {
     Assert.assertEquals(200, response.getStatusLine().getStatusCode());
     Assert.assertTrue(getResponseBody(response, true).contains(PLUGIN_NAME));
   }
+
+  @SneakyThrows
+  @Test
+  public void testGeoIpEnrichment() {
+    loadIndex(Index.GEOIP);
+    // Create a new dataSource
+    createDatasource(DATASOURCE_NAME, MANIFEST_LOCATION);
+    waitForDatasourceToBeAvailable(DATASOURCE_NAME, Duration.ofSeconds(10));
+
+    JSONObject resultGeoIp =
+            executeQuery(
+                    String.format(
+                            "search source=%s | eval enrichmentResult = geoip(\\\"%s\\\",%s)",
+                            TEST_INDEX_GEOIP, "dummycityindex", "ip"));
+
+    verifyColumn(resultGeoIp, columnName("name"), columnName("ip"), columnName("enrichmentResult"));
+    verifyDataRows(
+            resultGeoIp,
+            rows("Test user - USA", "10.1.1.1", Map.of("country", "USA", "city", "Seattle")),
+            rows("Test user - Canada", "127.1.1.1", Map.of("country", "Canada", "city", "Vancouver")));
+  }
+
+  /**
+   * Helper method to send a PUT request to create a dummy dataSource with provided endpoint for
+   * integration test.
+   *
+   * @param name Name of the dataSource
+   * @param properties Request payload in Json format
+   * @return Response for the create dataSource request.
+   * @throws IOException In case of network failure
+   */
+  private Response createDatasource(final String name, Map<String, Object> properties)
```

Comment:
Just use the constants...? 
```suggestion
  private Response createDatasource()
```